tests/acceptance/commercial/test_mender_gateway.py

```diff
@@ -44,7 +44,11 @@ def test_build_mender_gateway(
             request, prepared_test_build["build_dir"], "core-image*.ext4"
         )
 
-        output = subprocess.check_output(
-            ["debugfs", "-R", "stat /usr/bin/mender-gateway", image]
-        ).decode()
-        assert "Type: regular" in output
+        for file in [
```

Comment:
Super nit:
It is customary to  use `(tuples)` for constant use

Reply:
Fixed also for `test_monitor_addon.py`